Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -43,7 +45,7 @@ void ChessWidget::paint_event(GUI::PaintEvent& event)
     GUI::Painter painter(*this);
     painter.add_clip_rect(event.rect());
 
-    painter.fill_rect(frame_inner_rect(), Color::Black);
+    painter.fill_rect(frame_inner_rect(), Gfx::Color::WarmGray);
```

Comment:
There are a few other applications that draw things with a fixed aspect ratio and have a black background. `Flood` and `GameOfLife` are 2 examples I can think of off the top of my head. Personally, I don't have a strong preference either way, but I think it's good to be consistent.

If you keep this change, it should probably be split into a separate commit.